pkg/api/api.go

```diff
@@ -75,3 +83,17 @@ func (c *Client) Do(req *http.Request) (*http.Response, error) {
 
 	return resp, nil
 }
+
+func isLocalIPv6() bool {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
```

Comment:
Let's use `baseIPAddrv4` instead of `8.8.8.8` since the actual IP doesn't matter.